server/routes/middleware/Token.js

```diff
@@ -4,52 +4,58 @@ const User = require('../../models/User');
 var privateKey = "mySecret";
 
 var verifyToken = function (req, res, next) {
-  console.log("Verifying token.");
-  var token = req.body.token || req.query.token || req.headers['x-access-token'] || req.headers['authorization'].split(' ')[1]; // normal headers "Authorization: Bearer 2kj234df0ds2f3n40n"
-  if (!token)
-    return res.status(403).send({ auth: false, message: 'No token provided.' });
+  if (req.headers['authorization']) {
```

Comment:
This breaks versatility

Reply:
Ideally we should use x-access-token. Mention that to front end devs please.

Reply:
I really don't see how this is an issue, this is just another check on the same function. Many others had the same issue even after keeping requireRole() (no admin as the parameter) and would still get the error